tensorflow/models/image/cifar10/cifar10.py

```diff
@@ -392,3 +392,5 @@ def _progress(count, block_size, total_size):
     statinfo = os.stat(filepath)
     print('Successfully downloaded', filename, statinfo.st_size, 'bytes.')
     tarfile.open(filepath, 'r:gz').extractall(dest_directory)
```

Comment:
Can you just move this statement out of the if scope, so that repetition can be avoided?